source/standalone/tutorials/04_sensors/benchmark_cameras.py

```diff
@@ -0,0 +1,868 @@
+# Copyright (c) 2022-2024, The Isaac Lab Project Developers.
+# All rights reserved.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+
+"""
```

Comment:
Recommend moving this benchmarking to the benchmark folder instead of tutorials.